packages/darklang/languageTools/parser/core.dark

```diff
@@ -21,7 +21,7 @@ module Darklang =
 
           /// Where in the source code is this node written/contained
           /// i.e. Line 1, Column 2 to Line 1, Column 5
-          sourceRange: Range
+          range: Range
```

Comment:
Do we only need to rename this specific `sourceRange` label, or should we update all instances of sourceRanges,
( eg. `type SourceRange = Parser.Range`)?

Reply:
We can rename those SourceRange types to just Range as well, I think